purr/theme.json

```diff
@@ -0,0 +1,214 @@
+{
+	"settings": {
+		"appearanceTools": true,
+		"color": {
+			"palette": [
+				{
+					"color": "#f2f2f2",
+					"name": "Gray lightest",
+					"slug": "custom-gray-lightest"
+				},
+				{
+					"color": "#1da1f2",
+					"name": "Twitter blue",
```

Comment:
I'm not sure about using Twitter blue as the name of the color :D

Reply:
:D true. X blue?